### DISCUSSION TITLE
New Logo

### DISCUSSION THREAD

**Discussion by icarusinthewings** [vkbo/novelWriter#1517] [Ideas]
Hi!
I really love the novelWriter software. I think it would be great for the project to get a new logo. I've taken the liberty of designing one myself. Let me know any feedback that you have or if this is something you're interested in!
![NovelWriter logo Concept Sheet - Styles](https://github.com/vkbo/novelWriter/assets/24457862/3617a02c-4786-46d6-8e42-2b27d962dfab)
![novelWriter logo concept sheet - colors](https://github.com/vkbo/novelWriter/assets/24457862/3ec99550-0e5b-4414-b629-df77c8227374)
(The idea is to have adaptive logo colors for each theme, similar to Android 12's theme system.)
[Here's what it looks like running on the latest beta.](https://github.com/vkbo/novelWriter/assets/24457862/a40189f0-0ea3-4392-bd29-19ec5de59d6a)

**Comment by vkbo** [comment]
Oh, these look great!

Back when I created the current logo, I sent out a request on Twitter for help, and I didn't get much response. So I just made a basic one based on the Markdown logo.

I like the last row, with perhaps the blue one as the default? (sorry, my favourite colour.) I'm wondering if the book is more obviously a book if it has a slightly different tone on the spine, but maybe not as strong as on the first two on the top? What do you think?

**Reply by vkbo** [reply]
Also, on the "Open Project" dialog that you show in the clip, I do want to add a design on the left. I want to combine this dialog with the new project wizard into one dialog. The Wizard has the typewriter image on the left.

I have some idea of what kind of illustration I want. I think it was inspired by a Simpsons episode actually 😄

**Reply by xahodo** [reply]
Could the block in the background of the letter be made to resemble a book more?

Really like it.

nvm. Took a better look. Actually looks like a book already.

**Reply by vkbo** [reply]
I like the design of logo 2 from the top line, but was wondering what it'd look like with a smaller contrast between the spine and the cover of the book. Also, possibly with the spine being darker than the cover instead of lighter?

But yeah, it would be great to update the logo for version 2.x, and I think you really have something here. Thanks a lot for doing this!

**Reply by xahodo** [reply]
Perhaps the contrast between the different colors in the icon could be improved in the letter 'N' which ends up looking like a pencil?

**Reply by icarusinthewings** [reply]
Sorry for my late reply! My notifications have been messing up. So, what I'm hearing is you'd like something like #2 in blue, but with less contrast? I'm also looking at getting rid of the gap between the upward N stroke and the block.

**Reply by vkbo** [reply]
Yes, that's it. Just an idea, which may or may not work of course. I was just wondering how that would look 😄

**Reply by vkbo** [reply]
Hi! What's your time frame on this? I'm just wondering, because I have now started working on release 2.2, and hoped I maybe could update the icon here. If not, it can wait until 2.3 of course.

I want to make some design updates to the New Project Wizard and About dialog, and I'd like to add the logo and a design element to both these dialogs.

**Reply by vkbo** [reply]
I've been tweaking the current logos a bit in the mean time, and just realised we also need an icon for the project files themselves. Currently they look like this:

![application-x-novelwriter-project](https://github.com/vkbo/novelWriter/assets/1619840/4ee9de93-544f-4ffb-8b86-44c8d59efdee)

I'm thinking it could be the same as the main logo, but just in a different colour?

**Reply by HeyMyian** [reply]
I've tried to recreate the design @velocitydesign but couldn't quite get any part of it right in Illustrator. While I do have a formal education in media design, I just have to accept I am *not* a graphic designer. I find what they created visually appealing, which is why I did try. However, being visually appealing is not the end-all of logo design. For example, this [comment on Hacker News](https://news.ycombinator.com/item?id=27450041) has been living rent-free in my head since I've first read it. The proposed logo also has deeper implications if one were to unpack the symbols. The letter `N` is a pencil, but what a cumbersomely shaped pencil would that be? Try writing with that thing? It's got to be uncomfortable. It's not the best advertisement for a program that is designed to help people write.

I say all that, but I don't have a counter-offer. I'm going to attach my poorly attempted scribbles for posterity, but really, I got nothing.

[scribbles](https://github.com/vkbo/novelWriter/assets/60368695/2e331b27-449f-4351-92d9-220e2b8b6bef)

**Reply by vkbo** [reply]
Haha, well it says more about them than anyone else. It's based on the markdown logo. Anyway, it was only intended to be temporary until I could get a proper logo made, but here we are. It would be nice to be able to replace it for the upcoming 2.2 release. I could make the final changes (colour) myself if I had the vector file of what's already here.

**Reply by vkbo** [reply]
Honestly, I'm thinking about just making something as simple as a typewriter key with an "n" on it.

![Key](https://github.com/vkbo/novelWriter/assets/1619840/88d6f0f5-2647-4c98-9de9-2d1cbf97fa03)

**Reply by vkbo** [reply]
I still like the book though, so also tested this:

![Book256](https://github.com/vkbo/novelWriter/assets/1619840/32214013-27fb-4466-a267-31bd4668c015) ![Book128](https://github.com/vkbo/novelWriter/assets/1619840/bd244092-ddfc-4d9b-b8a8-60f1c1ff112a) ![Book64](https://github.com/vkbo/novelWriter/assets/1619840/fcab9e68-ecbd-4409-8894-d122ca1b081b) ![Book48](https://github.com/vkbo/novelWriter/assets/1619840/8971f962-d758-4494-84c7-cf6e9389339b)

**Reply by vkbo** [reply]
There, I'm done. I think I'll use these icons. Thanks for all the input everyone!

![novelwriter-256](https://github.com/vkbo/novelWriter/assets/1619840/a3dc137f-09eb-4614-bf13-aadd52d116a4) ![x-novelwriter-project-256](https://github.com/vkbo/novelWriter/assets/1619840/60284959-177e-4e95-9444-a03a563a02c9)

**Reply by peter88213** [reply]
Looks nice. but I think the monochrome version has some disadvantages with very small icon sizes. 
The original logo may not meet the highest aesthetic standards, but I find it funny for its phallic symbolism, and it has an immense practical advantage because it is clearly distinguishable. On the contrary, for example, the new Thunderbird logo, which looks confusingly similar to the Signal app icon in my narrow taskbar, where one blue blob sits next to the other.

Edit: 
Here is how it looks in 20 px size on my Windows taskbar: 

![taskbar](https://github.com/vkbo/novelWriter/assets/42072564/714a1207-7db4-4e02-85c8-b25604d6aff8)

**Reply by vkbo** [reply]
The previous logo was always meant to be temporary. I want a flatter look, and something that isn't just an icon. It also needs to work with other design elements like on the website. This one works nicely for both purposes. It also looks decent even in low resolution. Of course, 16x16 never looks very good.

I'm currently in dialog with an artist to create a splash screen image for the Open Project/Wizard/About boxes. So I also wanted something that can work on those when they've been redesigned.

**Reply by vkbo** [reply]
Also, the previous logo was a reference to Markdown, but novelWriter has drifted away from being a pure Markdown editor, and will do even more so in 2.2. I'm slowly removing references to Markdown in text descriptions as well, and instead emphasising plain text.

**Reply by peter88213** [reply]
> I'm slowly removing references to Markdown in text descriptions as well, and instead emphasising plain text.

What's the point of calling your markup "plain text" instead of "markdown"?

**Reply by vkbo** [reply]
Who said I was? I've stopped referring to *the application* as a "Markdown editor", because that has a lot of implications to people who know Markdown.

**Reply by icarusinthewings** [reply]
These look so good! I'm so happy that I was a part of this... I don't check Github often so I apologize that I wasn't as helpful!

**Reply by vkbo** [reply]
Thanks a lot for starting this and for your input and ideas 😃
